[2025-06-05T16:06:53.351+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-05T16:06:52.180028+00:00 [queued]>
[2025-06-05T16:06:53.376+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-05T16:06:52.180028+00:00 [queued]>
[2025-06-05T16:06:53.377+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-05T16:06:53.397+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2025-06-05 16:06:52.180028+00:00
[2025-06-05T16:06:53.404+0000] {standard_task_runner.py:60} INFO - Started process 53685 to run task
[2025-06-05T16:06:53.409+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'create_postgres_table', 'manual__2025-06-05T16:06:52.180028+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmpf55zt1o4']
[2025-06-05T16:06:53.411+0000] {standard_task_runner.py:88} INFO - Job 96: Subtask create_postgres_table
[2025-06-05T16:06:53.507+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table manual__2025-06-05T16:06:52.180028+00:00 [running]> on host 14aa2e8b0b7d
[2025-06-05T16:06:53.671+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T16:06:52.180028+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T16:06:52.180028+00:00'
[2025-06-05T16:06:53.673+0000] {sql.py:276} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2025-06-05T16:06:53.691+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:06:53.710+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:06:53.747+0000] {sql.py:450} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2025-06-05T16:06:53.777+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=create_postgres_table, execution_date=20250605T160652, start_date=20250605T160653, end_date=20250605T160653
[2025-06-05T16:06:53.824+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T16:06:53.885+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
